locust/stats.py

```diff
@@ -117,6 +176,19 @@ class StatsEntry(object):
     This dict is used to calculate the median and percentile response times.
     """
     
+    use_response_times_cache = False
+    """
+    If set to True, the copy of the response_time dict will be stored in response_times_cache 
+    every second, and kept for 20 seconds. We can use this dict to calculate the *current* 
```

Comment:
similar comment to my question about the 10 seconds sliding window time, I can see use cases for adjusting this (for example if your operations take a long time you may end up with wonky results here if the cache is only kept/reported for 20 seconds)

Reply:
I don't think it'll be more wonky than the other stats. Since the stats gets reported once a request is complete, if you have endpoints that takes really long (e.g. 60 seconds), the stats will feel kind of delayed.